### PR TITLE
onLoadComplete callback

### PR DESCRIPTION
**Pull request by kmathew-pdftron**
onLoadComplete callback Monday

### CODE REVIEW

File: API.md [hunk 1]
```diff
@@ -398,12 +398,30 @@ path | string | File path that the document has been saved to
...
+
+Parameters:
+
+Name | Type | Description
+--- | --- | ---
+path | string | File path that the document has been saved to
+
+```js
+<DocumentView
+  onDocumentLoaded = {(path) => {
```
Comment (sgong-pdftron) [inline API.md:420]: ?

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]

File: API.md [hunk 2]
```diff
@@ -398,12 +398,30 @@ path | string | File path that the document has been saved to
 
 ```js
 <DocumentView
-  onDocumentLoaded = {(path) => { 
+  onLoadComplete = {(path) => {
```
[the remark this answers is not in the document]
Comment (kmathew-pdftron) [reply API.md]: mistake fixed it

File: API.md (hunk 1 above)
Comment (kmathew-pdftron) [reply API.md]: mistake fixed

**Review by sgong-pdftron — APPROVED** [review]
no native code, you can merge this one without james

**Pull request merged**